paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerBottomSheet.kt

```diff
@@ -0,0 +1,57 @@
+package com.stripe.android.customersheet.ui
+
+import androidx.activity.compose.BackHandler
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.ModalBottomSheetLayout
+import androidx.compose.material.ModalBottomSheetValue
+import androidx.compose.material.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import kotlinx.coroutines.launch
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+internal fun CustomerBottomSheet(
+    modifier: Modifier = Modifier,
+    onClose: () -> Unit = {},
+    sheetContent: @Composable ColumnScope.() -> Unit,
+) {
+    val coroutineScope = rememberCoroutineScope()
+    val modalSheetState = rememberModalBottomSheetState(
+        initialValue = ModalBottomSheetValue.Hidden,
+        confirmValueChange = {
+            it != ModalBottomSheetValue.HalfExpanded
+        },
+        skipHalfExpanded = false
+    )
+    var sheetLaunched by remember { mutableStateOf(false) }
```

Comment:
This might need to be `rememberSaveable` to work across config changes, no?

Reply:
Also, is there anything on `ModalBottomSheetState` that allows us to keep track of this instead of a variable?

Reply:
> Also, is there anything on `ModalBottomSheetState` that allows us to keep track of this instead of a variable?

Not that I could find, no.